sys-auth/nss-mdns/files/lld-17-undefined-versioned-symbols.patch

```diff
@@ -0,0 +1,158 @@
+diff --git a/Makefile.am b/Makefile.am
```

Comment:
Can you add a link to the upstream PR + Gentoo bug at the top of the patch?

Reply:
done.